<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testsuite errors="0" failures="1" name="io.cucumber.core.plugin.JUnitFormatter" skipped="0" tests="1" time="29.105">
<testcase classname="User cannot login with invalid username validating the error message" name="US005TC06 Error message on blank screen when user signing in login page" time="28.258">
<failure message="" type="java.lang.AssertionError"><![CDATA[Given user is on “https://gmibank.com/login” page...........................passed
And the users cliks on Username.............................................passed
And the users enter the wrong Username......................................passed
And the users cliks on Password.............................................passed
And thee users enter thee wrong Password....................................passed
And the users clikson Sign in ..............................................passed
And the eror messaage onthe screeen is confirmed............................passed
And User clicks cancel after error message..................................passed
And If the user gets the error message, a bug has been found................failed

StackTrace:
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertFalse(Assert.java:65)
	at org.junit.Assert.assertFalse(Assert.java:75)
	at stepdefinitions.SignInStepDef.if_the_user_gets_the_error_message_a_bug_has_been_found(SignInStepDef.java:271)
	at ✽.If the user gets the error message, a bug has been found.(file:///Users/air/IdeaProjects/com.gmibank/src/test/resources/features/US05signin.feature:65)
]]></failure>
</testcase>
</testsuite>
